modules/core/src/main/java/org/apache/ignite/configuration/DataStorageConfiguration.java

```diff
@@ -308,7 +308,9 @@ public DataStorageConfiguration() {
 
     /**
      * Initial size of a data region reserved for system cache.
+     * Default value is {@link #DFLT_SYS_REG_INIT_SIZE}.
      *
+     * @see #setSystemRegionInitialSize(long).
```

Comment:
It seems that it’s not customary to write a dot after the see link

Reply:
Hi, @NSAmelchev ! I removed excess dots after JavaDoc see links and unnecessary whitespace in 485th line. Thank you!